erica_app/tests/request_processing/test_erica_input.py

```diff
@@ -221,6 +217,240 @@ def test_if_widowed_lived_separated_and_corresponding_date_then_raise_no_excepti
             pytest.fail("parse_obj failed with unexpected ValidationError " + str(e))
 
 
+class TestFormDataPersonAMerkzeichen:
+
+    def test_if_person_a_has_merkzeichen_bl_no_boolean_then_raise_validation_error(self, standard_est_data):
+        standard_est_data['person_a_has_merkzeichen_bl'] = 'yes'
+
+        with pytest.raises(ValidationError):
+            FormDataEst.parse_obj(standard_est_data)
+
+    def test_if_person_a_has_merkzeichen_tbl_no_boolean_then_raise_validation_error(self, standard_est_data):
+        standard_est_data['person_a_has_merkzeichen_tbl'] = 'yes'
+
+        with pytest.raises(ValidationError):
+            FormDataEst.parse_obj(standard_est_data)
+
+    def test_if_person_a_has_merkzeichen_h_no_boolean_then_raise_validation_error(self, standard_est_data):
+        standard_est_data['person_a_has_merkzeichen_h'] = 'yes'
+
+        with pytest.raises(ValidationError):
+            FormDataEst.parse_obj(standard_est_data)
+
+    def test_if_person_a_has_merkzeichen_ag_no_boolean_then_raise_validation_error(self, standard_est_data):
+        standard_est_data['person_a_has_merkzeichen_ag'] = 'yes'
+
+        with pytest.raises(ValidationError):
+            FormDataEst.parse_obj(standard_est_data)
+
+    def test_if_person_a_has_pflegegrad_no_boolean_then_raise_validation_error(self, standard_est_data):
+        standard_est_data['person_a_has_pflegegrad'] = 'yes'
+
+        with pytest.raises(ValidationError):
+            FormDataEst.parse_obj(standard_est_data)
+
+    def test_if_person_a_has_merkzeichen_bl_then_set_correct_field(self, standard_est_data):
+        standard_est_data['person_a_has_merkzeichen_bl'] = True
+
+        resulting_input_data = FormDataEst.parse_obj(standard_est_data)
+
+        assert resulting_input_data.person_a_has_merkzeichen_bl_tbl_h_pflegegrad is True
```

Comment:
Oh man, I feel so nit-picky saying this...buuuut: If a test has only three lines, can we omit the empty lines? I completely get why you add them and it makes sense to me whenever the test is a bit more complicated. But I can't confuse the setup-action-test here - as there are only three lines in total. 🙈

Reply:
To be honest, I was expecting you might say this. For me, it is still so much better to read with a little whitespace because I do not need to go through the lines to read the content. Somehow I can not get my brain to understand the order that good if they are in one block.

That said, I can change it. Did we ever make a final decision for how to handle this? If not, I think we might quickly have a discussion about it and settle this :)

Reply:
Haha, maybe that isn't something we all have to do the same way. I'll add it to the Dev Talk topics^^